internal/http/client.go

```diff
@@ -330,3 +290,35 @@ func makeRequestBodyReader(reqBody interface{}) (*bytes.Buffer, error) {
 
 	return b, nil
 }
+
+// GetBaseURL returns the BaseURL used by the client
+func (c *Client) GetBaseURL() string {
+	return c.config.BaseURL
+}
+
+// Query runs a graphQL query.
+func (c *Client) Query(query string, vars map[string]interface{}, respBody interface{}) error {
```

Comment:
Does this belong in `graphql.go` as well?

Reply:
Since it's a func on `Client` I would say no.